### DISCUSSION TITLE
Can't select a segment where the segment triangle is on the background layer

### DISCUSSION THREAD

**Discussion by thirdpoliceman** [Hypfer/Valetudo#1725] [Q&A / Support] [answered]
Hi,

Well done on such a nice piece of software. I have a small issue with the latest release (2023.02.0) on a S50 and was wondering if it is known or if there is a workaround. I can select any segment on my floorplan except for segments where the triangle is on the background layer (circled in red on image). The behavior exists on Chrome browser and Android App.

![segmentselect](https://user-images.githubusercontent.com/54640513/221680797-8be8b1ec-467f-4251-b16b-1eb0e89a1f2d.jpg)

Thank You.

**Comment by Hypfer** [comment, marked as answer]
Segment selection changed in 2022.11.
See the changelog here: https://github.com/Hypfer/Valetudo/releases/tag/2022.11.0

You can now just tap the orange segment pixels to select it.

**Reply by jcbtus** [reply]
:blush: Thank you very much. I can work with that.

Regards.
